libeos-updater-util/ostree-bloom.c

```diff
@@ -398,6 +398,12 @@ ostree_bloom_get_hash_func (OstreeBloom *bloom)
   return bloom->hash_func;
 }
 
+/* Ignore some warnings from the SipHash code rather than modifying it. */
+_Pragma ("GCC diagnostic push")
+_Pragma ("GCC diagnostic ignored \"-Wswitch-default\"")
+_Pragma ("GCC diagnostic ignored \"-Wimplicit-fallthrough\"")
```

Comment:
This causes a warning to print on gcc versions less than 7. That should be fine; just noting it.

Reply:
What warning? (I don’t have an old enough version of gcc to test with.) We should be able to add in a version check around the diagnostic line which is causing the warning.

Reply:
```
libeos-updater-util/ostree-bloom.c:404:1: warning: unknown option after ‘#pragma GCC diagnostic’ kind [-Wpragmas]
 _Pragma ("GCC diagnostic ignored \"-Wimplicit-fallthrough\"")
 ^~~~~~~
```

Reply:
Thanks. I’ve pushed an update which disables `-Wpragmas` too.